securedrop/sass/global/_body-tag.sass

```diff
@@ -0,0 +1,6 @@
+=body-tag
+  body
+    font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif
+    font-weight: 300
+    padding: 0
```

Comment:
I noticed this is not strictly copy/paste but 0px changed to 0 but ... whatever the unit zero still is zero ;-)

Reply:
Sorry, unconscious habit of mine to strip the unit off in CSS whenever it's a 0 (because as you say, the unit is redundant).